### PR TITLE
feat: Update videos

### PR DESCRIPTION
**Pull request by mohandast52**
## Proposed changes

- Video details like the link, thumbnail, etc., have been migrated to the CMS and removed from the UI. No further UI support is required (similar to how it’s handled for blogs).
- NOTE: We’ve also added a field for video (though highly recommended NOT using it and instead relying on reference links). It’s only intended for emergency/special use and supports files up to ~120MB. Also, when you click the link, the video will be downloaded rather than opened directly. If necessary, we can add support in the future to view the video in a page or modal.

## Screenshots/Recordings

- Loader

- Video

https://github.com/user-attachments/assets/c2c38165-658f-4d03-9030-2146616e77b2

## Things to keep in mind

- [ ] I confirm I have updated the meta title and description for the page

### CODE REVIEW

**Review by oaksprout — APPROVED** [review]

File: components/HomepageSection/Media.jsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+  const { videos, isLoading } = useFetchVideos(3);
+
+  return (
+    <SectionWrapper
+      backgroundType="SUBTLE_GRADIENT"
+      customClasses="px-4 md:px-8 py-12 lg:p-24 border-b"
+    >
+      <div className="max-w-screen-xl mx-auto flex flex-col gap-10">
+        <Videos isLoading={isLoading} videos={videos} limit={3} />
+        <Articles limit={3} tagFilter="bonds" showSeeAll />
```
[the remark this answers is not in the document]
Comment (atepem) [reply components/HomepageSection/Media.jsx:16]: I could probably add this into my current implement plausible branch. PR will (hopefully) be done tmrw

File: common-util/useFetchApi.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      title,
+      date,
+      platform_link,
+      drive_link,
+      platform,
+      thumbnail,
+      video: videoUploaded,
+    } = attributes || {};
+    const imageFilename = `${process.env.NEXT_PUBLIC_API_URL}${get(thumbnail, 'data.attributes.url') || ''}`;
+    const video_url = `${process.env.NEXT_PUBLIC_API_URL}${get(videoUploaded, 'data[0].attributes.url') || ''}`;
```
Comment (truemiller) [inline common-util/useFetchApi.js:38]: this urls would be wrong in the event that the `get` calls fail .. 

```suggestion
    const imageFilename = `${process.env.NEXT_PUBLIC_API_URL}${get(thumbnail, 'data.attributes.url') || ''}`;
    const video_url = `${process.env.NEXT_PUBLIC_API_URL}${get(videoUploaded, 'data[0].attributes.url') || ''}`;
```

File: components/Content/Articles.jsx [hunk 3]
```diff
@@ -8,9 +8,9 @@ import useSWR from 'swr';
 import { Spinner } from '../Spinner';
 import Article from './Article';
 
-export const URL = `${process.env.NEXT_PUBLIC_API_URL}/api`;
+const URL = `${process.env.NEXT_PUBLIC_API_URL}/api`;
```
Comment (truemiller) [inline components/Content/Articles.jsx:11]: same comment with `URL` overwriting node class/module

File: components/Content/Videos.jsx [hunk 4]
```diff
@@ -1,65 +1,91 @@
...
 
-        const isValidDateA = !isNaN(dateA.getTime());
-        const isValidDateB = !isNaN(dateB.getTime());
+const Video = ({ video }) => (
+  <a
+    href={video.platform_link || video.drive_link || video.video_url}
+    target="_blank"
+    rel="noopener noreferrer"
+  >
+    <article
```
Comment (truemiller) [inline components/Content/Videos.jsx:16]: article isn't right tag for this, use `section` or `div`

**Review by truemiller — APPROVED** [review]

**Pull request merged**